### PR TITLE
Add language support for rego (OpenPolicyAgent)

### PR DESCRIPTION
**Pull request by jcmuller**
Tree-sitter grammar from https://github.com/FallenAngel97/tree-sitter-rego.
Queries from https://github.com/nvim-treesitter/nvim-treesitter/blob/master/queries/rego/highlights.scm

### CODE REVIEW

File: runtime/queries/rego/injections.scm [hunk 1]
```diff
@@ -0,0 +1 @@
+(comment) @comment
```
Comment (the-mikedavis) [inline runtime/queries/rego/injections.scm:1]:
```suggestion
((comment) @injection.content
 (#set! injection.language "comment"))
```

Comment (jcmuller) [reply runtime/queries/rego/injections.scm]: Done.

**Review by the-mikedavis — APPROVED** [review]

**Comment by the-mikedavis** [comment]
Thanks! Could you add a note for regols to the language server wiki? https://github.com/helix-editor/helix/wiki/How-to-install-the-default-language-servers

**Comment by jcmuller** [comment]
Done!

**Pull request merged**